src/Resources/views/Search/results.html.twig

```diff
@@ -1,3 +1,5 @@
+{% sonata_template_deprecate 'none' %}
```

Comment:
why not put `results.html.twig`?

Reply:
Because there is no usage of this template 
https://github.com/sonata-project/ecommerce/search?q=results.html.twig&unscoped_q=results.html.twig

Reply:
I don't think so

```
{% include '@SonataProduct/Catalog/_product_grid.html.twig' %}
```

Reply:
I don't know really :man_shrugging: